### PR TITLE
Read func bh rc

### PR DESCRIPTION
**Pull request by bobobnj**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -28,12 +28,21 @@ import fish from './MockFish.js'
...
     <Switch>
     <Route exact path="/" component={Home} />
-    <Route path="/fishindex" component={FishIndex} />
+    <Route path="/fishindex" render={(props) => <FishIndex fish={this.state.fish}/>} />
+    
+          <Route
+            path="/fishshow/:id"
+            render={(props) => {
+              let id = props.match.params.id
+              let fish = this.state.fish.find(c => c.id === +id)
```
Comment (sjproctor) [inline src/App.js:42]: This code looks good, but I would be more clear about your local variable in the `.find()` method. Maybe changing `c` to `f` ?

File: src/pages/FishIndex.js [hunk 2]
```diff
@@ -1,11 +1,23 @@
 import React, { Component } from 'react'
+import { NavLink } from 'react-router-dom'
+// import { Card, CardTitle, Col } from 'reactstrap'
```
Comment (sjproctor) [inline src/pages/FishIndex.js:3]: If these imports are not being used they can be removed.

**Review by sjproctor — APPROVED** [review]
Just a couple comments to consider. Looks good!

File: src/pages/FishIndex.js (hunk 2 above)
Comment (bobobnj) [reply src/pages/FishIndex.js:3]: fixing in progress

File: src/App.js (hunk 1 above)
Comment (bobobnj) [reply src/App.js:42]: fix in progress

**Pull request merged**